dbms/src/Storages/DeltaMerge/RowKeyRange.h

```diff
@@ -260,7 +255,7 @@ using RowKeyValues = std::vector<RowKeyValue>;
 
 /// An optimized implementation that will try to compare IntHandle via comparing Int values directly.
 /// For common handles, per-byte comparison will be still used.
-inline int compare(const RowKeyValueRef & a, const RowKeyValueRef & b)
+inline int operator<=>(const RowKeyValueRef & a, const RowKeyValueRef & b)
```

Comment:
Should we make this return type as [`std::strong_ordering`](https://en.cppreference.com/w/cpp/utility/compare/strong_ordering)?